statsmodels/iolib/tableformatting.py

```diff
@@ -96,7 +96,7 @@
 fmt_params = copy.deepcopy(fmt_base)
 
 fmt3 = dict(
-            data_fmts = ["%s", "%s", "%8s", "%s", "%23s"],
+            data_fmts = ["%s", "%s", "%8s", "%s", "%12s"],
```

Comment:
if this is the format used for the params table, then I think it will need one extra value, i.e. add an additional `"%12s"`